docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +262,37 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view metrics in Dynatrace dashboards:
+
+- To view the status of the Dynatrace integration with the Kyma Telemetry module, import the file [Telemetry Module Status](./telemetry-resource-metrics.json) as a Dynatrace dashboard. For details on how to import a dashboard, see [Importing Dashboards](https://docs.dynatrace.com/docs/analyze-explore-automate/dashboards-classic/dashboards/dashboard-json#import-dashboard).
+> [!WARNING]
+> Kyma Telemetry resources contain the following custom OpenTelemetry resource attributes:
+> - `k8s.resource.name`
+> - `k8s.resource.group`
+> - `k8s.resource.kind`
+> - `k8s.resource.version`
+> 
+> For the dashboard to function properly, you must [add these attributes to the allow list of OpenTelemetry metrics resource attributes](https://docs.dynatrace.com/docs/ingest-from/opentelemetry/getting-started/metrics/configuration#allow-list).
+
+## Use Dynatrace Alerts
+
+You can create Dynatrace Alerts based on certain metric events to send alerts about the Kyma Telemetry module status to your system of choice. 
+
+1. **Create a problem alerting profile**  
+   Define how and when alerts should trigger. Follow the [Dynatrace guide](https://docs.dynatrace.com/docs/analyze-explore-automate/notifications-and-alerting/alerting-profiles#create-an-alerting-profile).
```

Comment:
```suggestion
   Define how and when alerts should trigger. For details, see [Create an alerting profile](https://docs.dynatrace.com/docs/analyze-explore-automate/notifications-and-alerting/alerting-profiles#create-an-alerting-profile).
```

I like specific link titles (they increase the chance that a reader can find the target document even if the link is broken)